layout-server/src/main/java/org/finos/vuu/layoutserver/utils/DefaultApplicationLayoutLoader.java

```diff
@@ -0,0 +1,40 @@
+package org.finos.vuu.layoutserver.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.finos.vuu.layoutserver.exceptions.InternalServerErrorException;
+import org.finos.vuu.layoutserver.model.ApplicationLayout;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+public class DefaultApplicationLayoutLoader {
+    private static final String DEFAULT_LAYOUT_FILE = "defaultApplicationLayout.json";
+    private static ApplicationLayout defaultLayout;
+
+    @Bean
+    public ApplicationLayout getDefaultLayout() {
+        if (defaultLayout == null) {
+            loadDefaultLayout();
+        }
+        return defaultLayout;
+    }
```

Comment:
I was sat here thinking "I wonder why we've '@Bean'ed this rather than a static method. But, I'd forgotten the whole purpose of extracting this into a class was to make it mockable in tests. So the `@Bean` makes perfect sense. I wonder if there's any other advantange to Spring-ifying this with a `@Bean` rather than using a static method.

Reply:
As you say, I think facilitating tests is reason enough to use this pattern. I suppose in theory, this could also give us the option of having different, configurable options for loading a default layout.